ansible/config_sonic_basedon_testbed.yml

```diff
@@ -730,6 +730,20 @@
         become: true
         shell: config bgp startup all
 
+      - name: Copy smartswitch dpu config
+        copy: src=golden_config_db/smartswitch_dpu_extra.json
+              dest=/tmp/dpu_extra.json
+        become: true
+        when: topo in ["t1-smartswitch-ha","t1-28-lag","smartswitch-t1"]
```

Comment:
>t1-28-lag [](http://example.com/codeflow?start=44&length=9)

Is it possible that "t1-28-lag" could be a non-smart-switch topo?

Reply:
If yes, we may need to check other indicator of SmartSwitch.

Reply:
No. It is smartswitch only.

Reply:
Please fix in future PR [17773](https://github.com/sonic-net/sonic-mgmt/issues/17773)